usaspending_api/database_scripts/broker_matviews/transaction_matview.sql

```diff
@@ -0,0 +1,262 @@
+drop materialized view if exists transaction_matview;
+
+create materialized view transaction_matview as
+(
+select
+    *,
+    fy(action_date) as fiscal_year
```

Comment:
is fy() already defined by django migrations?

Reply:
This is in `common/helpers.py` as `FY_PG_FUNCTION_DEF`. It's run as part of the initial migration in the `accounts` django app

Reply:
👍